pkg/monitor/cluster/cluster.go

```diff
@@ -157,6 +180,11 @@ func (mon *Monitor) emitGauge(m string, value int64, dims map[string]string) {
 	mon.m.EmitGauge(m, value, dims)
 }
 
+func (mon *Monitor) logAndEmitError(fName string, err error) {
+	mon.log.Printf("%s: %s", fName, err)
+	mon.emitGauge("monitor.clustererrors", 1, map[string]string{"monitor": fName})
+}
```

Comment:
Pulled this into a function because it was going to be needed in a third place in this file. There may be a better location for this function definition.